src/matlab2tikz.m

```diff
@@ -1872,10 +1872,12 @@ function assertRegularAxes(handle)
     lineOpts = opts_new();
 
     % Get the options from the handle
-    lineStyle = get(h, 'LineStyle');
     lineWidth = get(h, 'LineWidth');
 
-    if ~isNone(lineStyle) && (lineWidth > m2t.tol)
+    % Get the line style and check whether it is the default one
+    [lineStyle, isDefaultLS] = getAndCheckDefault('Line', h, 'LineStyle', '-');
```

Comment:
This is for some reason returning false and causing many tests to fail because it adds the `solid` option:

```
test15-converted.tex
59c59
< \addplot [color=mycolor1,solid,forget plot]
---
> \addplot [color=mycolor1,forget plot]
```